src/jsx/relax.jsx

```diff
@@ -205,16 +206,43 @@ class RELAX extends React.Component{
     data["trees"] = {
       branchLengths: _.mapObject(data.fits, (model_val, model_key) => {
         return _.mapObject(data['branch attributes'][0], (branch_val, branch_key) => {
-          return branch_val[model_key]; 
+          return branch_val[model_key];
         });
       })
     }
-   
+
     _.keys(data.fits).forEach(model=>{
       data["fits"][model]["branch-annotations"] = this.formatBranchAnnotations(data, model);
       data["fits"][model]["annotation-tag"] = model == "MG94xREV with separate rates for branch sets" ? "ω" : 'k';
     });
 
+    /*
+    Data munge for the branch attribute table.
+      Adds two arrays to RELAX state: an array for headers and an array for rows.
+      These arrays will be consumed by the DatamonkeyTable component.
+    */
+    // Step 1: Get individual branch information from JSON sources.
+    var branchAttributes = data["branch attributes"][0];
+    var branchTestedStatuses = data["tested"][0];
+    var branchLengthsGTR = data["trees"]["branchLengths"]["Nucleotide GTR"];
+    // Step 2: Combine the info into one object.
+    var branchAttributesCombined = {};
+    for (var key in branchAttributes){
+      branchAttributesCombined[key] = {"Branch name": key, "Partition": branchTestedStatuses[key], "Branch length": branchLengthsGTR[key]};
```

Comment:
For some reason, this is throwing an error on the CD2 and Flu datasets.

Reply:
It looks like neither of these files has a "trees" object. The data sets look like they were uploaded in October 2017 and the HyPhy JSON fields were updated in December 2017 (http://hyphy.org/resources/json-fields.pdf). I ran a test example on Datamonkey.org (http://datamonkey.org/relax/5a820007e7b8b163187b7005) and the JSON does have a trees file. 

Let me know your thoughts but I think this PR should be good to review now and we should open a new issue to update the data that comes with HyPhy-Vision to match the new JSON format.

Reply:
@stephenshank: As discussed, I re-ran the CD2 and Flu datasets on datamonkey.org and used the results to replace the existing JSON files in methods/RELAX/data.